src/utils/htoken.ts

```diff
@@ -28,3 +30,26 @@ function getRatingScore(hToken: hToken, officialFR: string) {
 
   return creatorScore + depositorScore + feeRecipientScore
 }
+
+/**
+ * @param {hToken} hToken pool hToken
+ * @param {BigNumber} depositAmount new deposit amount to calculate the APY for
+ * @returns {BigNumber} pool APY for given deposit amount
+ */
+export function getPoolApy(hToken: hToken, depositAmount: BigNumber): BigNumber {
+  const poolBalance = toTokenAmount(hToken.tokenBalance, hToken.decimals)
+  const poolDuration = hToken.expiry - hToken.createdAt
```

Comment:
Which one do you think make more sense:
1. (current implementation) `duration = hToken.expiry - hToken.createdAt`
2. `duration = hToken.expiry - (Date.now() / 1000)`
calculate with 2 give us a higher APY as we approach expiry, while (1) will stay still.

I was originally thinking of (2), and also consider the share you get will get lower and lower as time goes by. 
But using (1) and assuming constant shares might actually be a good estimation as well. (as using duration will slightly underestimate the number, but using deposit amount instead of shares will slightly overestimate the number, they might just cancel off.)